Hunting Queries/AuditLogs/RareAuditActivityByApp.yaml

```diff
@@ -19,35 +19,37 @@ query: |
   let current = 1d;
   let auditLookback = 14d;
   let AuditTrail = AuditLogs 
```

Comment:
May want to add Global AllowList for Known Operations by Legit App occurring less frequent and classifying as rare.

Saw benign noise with below combination

- InitiatedByApp == "Microsoft Azure AD Group-Based Licensing"
- OperationName == "Change user license"

Reply:
done.